src/Components/SocialLogin.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { GoogleLogin } from 'react-google-login';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
+import Button from './Common/Button/Button';
+import { socialLogin } from '../actions/socialLogin';
+
+/**
+ * @param {*} props
+ * @returns {object} jsx
+ * Social login component
+ */
+export class SocialLogin extends Component {
+  callTwitterLogin = () => {
+    window.location.replace(`${process.env.REACT_APP_BACKEND_URL}/api/auth/twitter`);
+  };
+
+  responseSocialLogin = ({ response, provider }) => {
+    const { onSocialLogin } = this.props;
+    const { accessToken } = response;
+    onSocialLogin(accessToken, provider);
+  };
+
+  /**
+   * @returns {*} dispatch
+   */
+  render() {
+    return (
+      <>
```

Comment:
I don't get what these tags are doing

Reply:
These tags are just serving as wrapper. They replace React.Fragment.

Reply:
It is a new syntax which is not supported by many tools and react documentation advise to use `React.fragment` instead. For more information, you can check this [link](https://reactjs.org/docs/fragments.html)

Reply:
Cool...